easybuild/tools/config.py

```diff
@@ -865,6 +867,46 @@ def log_path(ec=None):
     return log_file_format(return_directory=True, ec=ec, date=date, timestamp=timestamp)
 
 
+def get_failed_installs_build_dirs_path(ec):
+    """
+    Return the 'failed_installs_build_dirs_path',
+    the location where build directories are copied if installation failed
+
+    :param ec:  dict-like value with 'name' and 'version' keys defined
+    """
+    failed_installs_build_dirs_path = ConfigurationVariables()['failed_installs_build_dirs_path']
+
+    if not failed_installs_build_dirs_path:
```

Comment:
How about making it a bit shorter and clear that this is not the final path yet?
```suggestion
    base_path = ConfigurationVariables()['failed_installs_build_dirs_path']
    if not base_path:
```